docs/_articles/en/user-guide/source-diff.md

```diff
@@ -3,33 +3,12 @@ title: Source Diff
 lang: en
 ---
 
-The Source Diff command allows you to diff metadata types against your default org. This feature makes it easier to visualize the changes between your local project and the metadata in your org.
-
-> NOTICE: The Source Diff feature is currently in beta. If you find any bugs or have feedback, [open a GitHub issue](./en/bugs-and-feedback).
-
-## Setup
-
-Because the Source Diff feature is in beta, you must install a Salesforce CLI plugin. From the terminal, run `sfdx plugins:install @salesforce/sfdx-diff`.
-After the installation is complete, `@salesforce/sfdx-diff` appears in the list of installed plugins when running `sfdx plugins`.
+The Source Diff command allows you to diff metadata types against your default org. This feature makes it easier to visualize the changes between files in your local project and the metadata in your org. You can look at Diffs for an individual file or directory..
 
 ## Usage
+Menu options `SFDX: Diff File Against Org` and `SFDX: Diff Folder Against Org` appear when you right-click a file or folder.  
 
-Starting in version `46.11.0`, a new menu option `SFDX: Diff File Against Org` appears when you right-click an open metadata file.
-
-![Source Diff command](./images/source_diff.png)
-
-Source Diff currently supports the following metadata:
+![Source Diff command](../../../images/source_diff.png)
 
-- Apex Class
-- Apex Trigger
-- Aura Application
-- Aura Component
-- Aura Event
-- Aura Interface
-- Aura Token
-- Custom Label
-- Layout
-- Lightning Web Component
-- Permission Set
-- Visualforce Component
-- Visualforce Page
+You can then view individual file diffs side by side. If a file doesn’t exist locally or in the Org, it will not show up in the list.
```

Comment:
Do we want to explicitly say that Diff/Conflict Detection now should support all metadata types?

Reply:
I don't think we need to explicitly doc it.  Hopefully the lack of caveats will clue folks in that they can use it for everything.  It would be good to mention in the changelog notes though.